src/main/java/baritone/command/defaults/ToMapCommand.java

```diff
@@ -0,0 +1,223 @@
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Color averageColor(BufferedImage bi, String id) {
+        if (!cachedColors.containsKey(id)) {
+//            System.out.println(id);
+            int x1 = 16;
+            int y1 = 16;
+            long sumr = 0, sumg = 0, sumb = 0;
+            for (int x = 0; x < x1; x++) {
+                for (int y = 0; y < y1; y++) {
+                    Color pixel = new Color(bi.getRGB(x, y));
+                    sumr += pixel.getRed();
+                    sumg += pixel.getGreen();
+                    sumb += pixel.getBlue();
+                }
+            }
+            int num = 16 * 16;
+            try {
+                cachedColors.put(id, new Color(sumr / num, sumg / num, sumb / num));
+            } catch (Exception e) {
+                cachedColors.put(id, new Color(Color.TRANSLUCENT));
+            }
+        }
+        return cachedColors.get(id);
+    }
+
+    private static BufferedImage getTextureForBlockId(String blockId) throws IOException {
+        String base = "/home/nexor/.minecraft/versions/1.15.2-OptiFine_HD_U_G5_pre1/1.15.2-OptiFine_HD_U_G5_pre1/assets/minecraft/textures/block/";
+        Path withTopPath = Paths.get(base + blockId + "_top.png");
+        Path withoutTopPath = Paths.get(base + blockId + ".png");
+        Path debug = Paths.get(base + "debug.png");
+        if (Files.exists(withTopPath)) {
+            return ImageIO.read(Files.newInputStream(withTopPath));
+        }
+        if (Files.exists(withoutTopPath)) {
+            return ImageIO.read(Files.newInputStream(withoutTopPath));
+        }
+        return ImageIO.read(Files.newInputStream(debug));
+    }
+
+    public static Map<String, int[]> getBlockCoordsOfRegionCorners(int x, int z) {
+        return new HashMap<String, int[]>() {
+            {
+                put("NW", new int[]{
+                        x * 512,
+                        z * 512,
+                });
+                put("NE", new int[]{
+                        (x + 1) * 512 - 1,
+                        z * 512,
+                });
+                put("SW", new int[]{
+                        x * 512,
+                        (z + 1) * 512 - 1,
+                });
+                put("SE", new int[]{
+                        (x + 1) * 512 - 1,
+                        (z + 1) * 512 - 1,
+                });
+            }
+        };
+    }
+
+    // http://jens-na.github.io/2013/11/06/java-how-to-concat-buffered-images/
+    // /home/nexor/.minecraft/versions/1.15.2-OptiFine_HD_U_G5_pre1/1.15.2-OptiFine_HD_U_G5_pre1/assets/minecraft/textures/block
+    private void concatImages() {
+
+    }
+
+    @Override
+    public Stream<String> tabComplete(String label, IArgConsumer args) throws CommandException {
+        return Stream.empty();
+    }
+
+    @Override
+    public String getShortDesc() {
+        return "null";
```

Comment:
maybe put something more informative here

Reply:
The issue is informative enough for now

Reply:
This is the ingame help, for those that didn't read the issue (hopefully that is everyone soon)
I personally think it's better to do stuff like this before forgetting it, but maybe you are better in remembering stuff than I am

Reply:
I didn't even stop to think about a description. I think this should be the last thing to be written to this file.